purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1313,10 +1317,16 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
                 platformInfo,
                 !observerMode
             )
+
+            val httpClient = proxyURL?.let {
+                debugLog("Purchases is being configured using a proxy for RevenueCat")
+                HTTPClient(appConfig, baseURL = it)
+            } ?: HTTPClient(appConfig)
```

Comment:
left some thoughts in a comment that disappeared because there were file changes after that, but they should be visible from the conversation view in the PR.